custom.html.md.erb

```diff
@@ -143,6 +143,60 @@ $ cf create-buildpack BUILDPACK PATH POSITION [--enable|--disable]
 </pre>
 
 <%=vars.link_adminguid_buildpack%>
+### <a id='specifying-default-versions'></a> Specifying Default Versions ###
+As of `buildpack-packager` [version 2.3.0](https://github.com/cloudfoundry/buildpack-packager/releases/tag/v2.3.0) and later, it is possible to specify the default version for a given dependency. This is done by adding a `default_versions` object to the `manifest.yml` file.
```

Comment:
Versioning is tricky in docs, I'd prefer we'd be a bit noisier and at the same time, more up front.